packages/phrases-ui/src/locales/zh-cn.ts

```diff
@@ -81,10 +81,12 @@ const translation = {
     link_email: '绑定邮箱',
     link_phone: '绑定手机',
     link_email_or_phone: '绑定邮箱或手机号',
-    link_email_description: '绑定邮箱以保障您的账号安全',
-    link_phone_description: '绑定手机号以保障您的账号安全',
-    link_email_or_phone_description: '绑定邮箱或手机号以保障您的账号安全',
-    continue_with_more_information: '为保障您的账号安全，需要您补充以下信息。',
+    link_email_description: '绑定邮箱以保障你的账号安全',
+    link_phone_description: '绑定手机号以保障你的账号安全',
+    link_email_or_phone_description: '绑定邮箱或手机号以保障你的账号安全',
+    continue_with_more_information: '为保障你的账号安全，需要你补充以下信息。',
```

Comment:
nit: 
```suggestion
    continue_with_more_information: '为保障账号安全，需补充以下信息。',
```
Also, why this sentence ends with a period `。` while the others don't?

Reply:
might be legacy issue we have a separate PR to refine all these contents.